src/ddmd/dsymbolsem.d

```diff
@@ -62,6 +62,7 @@ import ddmd.utils;
 import ddmd.semantic;
 import ddmd.statement;
 import ddmd.target;
+import ddmd.templateparamsem;
 import ddmd.visitor;
```

Comment:
How about starting to use local, selective imports?
This would also provide the additional benefit that the code is easier readable ;-)

Reply:
Let's stick to this PR in this thread.